ci/awsClient.sc

```diff
@@ -39,9 +39,14 @@ case class Artifact(path: S3Path, sha1: String) {
   def downloadUrl: String = s"$base/${path.bucket}/${path.key}"
 }
 
-val S3_PREFIX = S3Path(
-  sys.env.getOrElse("S3_BUCKET", "downloads.mesosphere.io"),
-  Path(sys.env.getOrElse("S3_PATH" , "/marathon/snapshots")))
+/**
+ * builds = snapshots / builds, milestones, releases
```

Comment:
Why `/ builds`?

Why not
```
/**
  * @return S3 path to buildLocation in S3 bucket
  */
```

Reply:
It is not clear what is being asked.     Our release process: https://docs.google.com/document/d/1BRFbCjgu0N4nwr4f9daxJLIr_j9nqI19u5CAS-0jQio/edit#
indicates that we will have regular master builds with have been called snaps, snapshots and not pre-X, which currently land in a snapshots directory.   We are going to move those to `builds` directory on S3.   Planning to get rid of snapshots completely.   the word carries meaning which is not true for us.

Reply:
That was the missing information. I'd make it `builds` then. No more `snapshots`.